Update admins.xml

lazyknob opened the pull request.

lazyknob committed:
Update admins.xml

lazyknob committed:
Update admins.xml

abshkd commented:
KIV this for now.

abshkd reviewed (changes requested):
try to update your fork with my latest changes for future edits. I have manually merged this small edit for now

lazyknob committed:
Update admins.xml

Removed Shutdown command (too risky if server shutsdown and I cannot turn it back on?)

Added lps and lpb commands
